### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by sind14**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,3 +1,57 @@
...
+    def __init__(self, km: Any) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Any) -> Any:
```
Comment (Bohdan-Salamakha) [inline app/main.py:14]: other can be int, float, Distance
return type is Distance

File: app/main.py [hunk 2]
```diff
@@ -1,3 +1,57 @@
...
+        if isinstance(other, (int, float)):
+            return Distance(self.km + other)
+        return Distance(self.km + other.km)
+
+    def __iadd__(self, other: Any) -> Any:
```
Comment (Bohdan-Salamakha) [inline app/main.py:19]: other can be int, float, Distance
return type is Distance

File: app/main.py [hunk 3]
```diff
@@ -1,3 +1,57 @@
...
+        if isinstance(other, (int, float)):
+            self.km += other
+        else:
+            self.km += other.km
+        return self
+
+    def __mul__(self, other: Any) -> Any:
```
Comment (Bohdan-Salamakha) [inline app/main.py:26]: other can be int, float, Distance
return type is Distance

File: app/main.py [hunk 4]
```diff
@@ -1,3 +1,57 @@
...
+        if isinstance(other, (int, float)):
+            return Distance(self.km * other)
+
+    def __truediv__(self, other: Any) -> Any:
```
Comment (Bohdan-Salamakha) [inline app/main.py:30]: other can be int, float, Distance
return type is Distance

File: app/main.py [hunk 5]
```diff
@@ -1,3 +1,57 @@
...
+
+    def __le__(self, other: Any) -> Any:
+        if isinstance(other, (int, float)):
+            return self.km <= other
+        return self.km <= other.km
+
+    def __ge__(self, other: Any) -> Any:
+        if isinstance(other, (int, float)):
+            return self.km >= other
+        return self.km >= other.km
```
Comment (Bohdan-Salamakha) [inline app/main.py:57]: other can be int, float, Distance
return type is bool

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
v1

**Review by EdAlekseiev — APPROVED** [review]
Good job

**Pull request closed without merging**